src/openrct2-ui/windows/EditorInventionsList.cpp

```diff
@@ -435,13 +421,14 @@ static void window_editor_inventions_list_update(rct_window* w)
  */
 static void window_editor_inventions_list_scrollgetheight(rct_window* w, int32_t scrollIndex, int32_t* width, int32_t* height)
```

Comment:
Can't this simply return int32_t instead of having a pointer passed?

Reply:
width can be also dropped doesn't seem to be used

Reply:
This is a window event, you can't just change the signature. You'd have to change every window.